src/app/public/modules/text-editor/defaults/font-list-defaults.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  SkyTextEditorFont
+} from '../types/font-state';
+
+export const FONT_LIST_DEFAULTS: SkyTextEditorFont[] = [
```

Comment:
Is this internal? If so - we need to add that JS Doc

Reply:
Yep. Updated.